packages/clerk-js/src/ui/components/ApiKeys/ApiKeys.tsx

```diff
@@ -183,19 +205,17 @@ export const APIKeysPage = ({ subject, perPage, revokeModalRoot }: APIKeysPagePr
         elementDescriptor={descriptors.apiKeysTable}
         canManageAPIKeys={(isOrg && canManageAPIKeys) || !isOrg}
       />
-      {itemCount > (perPage ?? 5) && (
+      {(itemCount > (perPage ?? 5) || pageCount > 1) && (
```

Comment:
🙃 Seems the page size in `useApiKeys` is 10. Maybe we set a const there and reference it here?